### PR TITLE
add scripts to fix sufficients inconsistency for accounts

### PR DESCRIPTION
**Pull request by nbaztec**
Adds a script to create scenarios with broken accounts and a followup script to fix them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "@polkadot/util-crypto": "^8.7.1",
     "chalk": "^4.1.2",
     "esbuild": "^0.13.8",
+    "lodash.chunk": "^4.2.0",
```
Comment (crystalin) [inline package.json:16]: I think loading lodash just for chunk seems a bit too much no ?

File: src/scenarios/create-invalid-accounts.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+/*
```
Comment (crystalin) [inline src/scenarios/create-invalid-accounts.ts:1]: maybe create-accounts-with-invalid-providers.ts to be a bit more explicit ?

File: src/utils/functions.ts [hunk 3]
```diff
@@ -9,4 +14,30 @@ export const promiseConcurrent = <T, R>(
 
 export const numberWithCommas = (x: number | bigint | string) => {
   return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
-}
\ No newline at end of file
+}
+
+export async function waitTxDone(api: ApiPromise, tx: SubmittableExtrinsic, sender: KeyringPair, options?: Partial<SignerOptions>): Promise<string> {
```
Comment (crystalin) [inline src/utils/functions.ts:19]: This should be part of the transactions.ts

Comment (crystalin) [reply src/utils/functions.ts:19]: Also it might already exist there.

File: package.json (hunk 1 above)
Comment (nbaztec) [reply package.json:16]: This is only importing `lodash.chunk`, so not the entire library. Should be lite imo.

File: src/utils/functions.ts (hunk 3 above)
Comment (nbaztec) [reply src/utils/functions.ts:19]: I noticed the `sendAllStreamAndWaitLast` function however, it wasn't  giving the necessary events for debugging, as specified here: https://polkadot.js.org/docs/api/cookbook/tx/#how-do-i-get-the-decoded-enum-for-an-extrinsicfailed-event
It also seemed quite specific for a pre-defined use case. 

Does it make sense to align both these functions to be one?

Comment (crystalin) [reply src/utils/functions.ts:19]: If it is not disruptive yes, otherwise you can keep separate for now

Comment (nbaztec) [reply src/utils/functions.ts:19]: I gave it a shot here https://github.com/PureStake/moonbeam-tools/pull/8

did not test it locally though.

**Comment by nbaztec** [comment]
No longer required.

**Pull request closed without merging**